book/scheduling.md

```diff
@@ -1016,8 +1015,9 @@ is really being spent rendering. It's important to always measure
 before optimizing, because the result is often surprising.
 
 To instrument our browser, let's have it output the
-[JSON][json] tracing format used by [chrome://tracing][chrome-tracing]
-in Chrome, [Firefox Profiler](https://profiler.firefox.com/) or
+[JSON][json] tracing format used by
```

Comment:
Why make this change? Seems not significant.

Not critical in any case

Reply:
Oh yeah, it looks weird to say "foo bar [12] in Chrome, Foo Bar [13], Foo Bar [14]". If we move "in Chrome" earlier the reference-comma becomes a consistent pattern and helps parsing because the references are big and commas are small.

Reply:
Not critical.